messaging/src/test/java/org/axonframework/eventhandling/tokenstore/jdbc/JdbcTokenStoreTest.java

```diff
@@ -116,6 +118,70 @@ void testUpdateAndLoadNullToken() {
         assertNull(token);
     }
 
+    @Transactional
+    @Test
+    void testFetchTokenBySegment() {
```

Comment:
Nit: I am missing how the tests `testFetchTokenBySegment` and `testFetchTokenBySegmentSegment0` expect the result to not exist.

Reply:
Isn't this the default behaviour, since the token does not exist yet? I thought we'd first have to manually update the token (for example after processing a first event), before this method would return a token, or am I missing something?. What I mainly wanted to test here is that there are no issues in claiming the segment, which would cause an `UnableToClaimSegmentException` to be thrown and the test to fail. I could remove the `assertNull` call and just ignore the result of the `tokenStore.fetchtoken` call. Assuming my above understanding is correct, that should result in testing the same thing I want to test here.

Reply:
I am putting the JavaDoc next to this right now, and I couldn't agree more. Thanks for pointing this out, @Vermorkentech.